ci/daily_tell_slack.yml

```diff
@@ -16,7 +16,9 @@ steps:
         MESSAGE=${{ parameters['success-message'] }}
     fi
     PAYLOAD="{\"text\":\"$MESSAGE\n\"}"
-    if [ "$(variables['Build.SourceBranchName'])" = "master" ]; then
+    [ -n "$(Build.SourceBranchName)" ]
```

Comment:
Probably wanna delete this line now.

Reply:
I’ve just gone back to my first version now.